TESTING.md

```diff
@@ -0,0 +1,143 @@
+# How to run phpunit tests
+
+For reference: https://propelorm.org/Propel/cookbook/working-with-unit-tests.html
+
+### 1. Use separate containers because this package requires own database to test its functionality.
+
+Minimal docker configuration for tests:
+
+`docker-compose.yml:`
+
+```yml
+services:
+  php:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./src:/var/www
+    depends_on:
+      - mysql
+    networks:
+      - app-network
+    command: tail -f /dev/null
+
+  mysql:
+    image: mysql:8.0
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: php7
+      MYSQL_USER: user
+      MYSQL_PASSWORD: password
+    volumes:
+      - mysql-data:/var/lib/mysql
+    networks:
+      - app-network
+
+networks:
+  app-network:
+    driver: bridge
+
+volumes:
+  mysql-data:
+```
+
+`.env:`
+
+```dotenv
+COMPOSE_PROJECT_NAME=propel1-test-php74
```

Comment:
what is the purpose of this env variable ?

Reply:
it is for adding a prefix to all container names
to avoid mixing names
but it can be done with docker-compose, so i will remove this one